aiida_mlip/helpers/converters.py

```diff
@@ -99,3 +100,41 @@ def convert_to_nodes(dictionary: dict, convert_all: bool = False) -> dict:
             continue
         new_dict[key] = value
     return new_dict
+
+
+def kwarg_to_param(params: dict[str, Any]) -> list[str]:
+    """
+    Convert a dictionary of kwargs to a set of commandline flags.
+
+    Bools are converted as though ``store_true`` flag keys.
+
+    Parameters
+    ----------
+    params : dict[str, Any]
+        Dictionary of arguments to convert.
+
+    Returns
+    -------
+    list[str]
+        Commandline arguments as flags.
+
+    Examples
+    --------
+    >>> kwarg_to_param({"name": "Geoff", "key": True})
+    ['--name', 'Geoff', '--key']
+    >>> kwarg_to_param({"value": 6, "falsey": False})
+    ['--value', '6']
+    """
+    cmdline_params = []
+
+    for key, val in params.items():
+        key = key.replace("_", "-")
+        match val:
+            case bool() if val:
+                cmdline_params.append(f"--{key}")
+            case bool():
+                ...
```

Comment:
I may just not be thinking this through enough, but doesn't this assume the default is `False`?

If the default is `True`, we need to set the `--no-key` flag, which I don't think this does?

Reply:
Are there any parameters like this? Are all parameters compatible with `--no-key`?

Reply:
I'm not certain there are any currently settable in aiida-mlip, but there are certainly are few in janus e.g. `janus eos` defaults to `--minimize`, and everything currently defaults to `--tracker` (which we probably need to make sure we can disable).

If it's a boolean, it would always be `--no-key`, I believe. We just define the `key`, and it automatically creates the two flags. We probably could overwrite this in theory, but I don't think we do.